app/bundles/LeadBundle/Config/config.php

```diff
@@ -414,6 +414,14 @@
                 'tag'   => 'validator.constraint_validator',
                 'alias' => 'segment_in_use',
             ],
+            \Mautic\LeadBundle\Validator\Constraints\SegmentDateValidator::class => [
+                'class'     => \Mautic\LeadBundle\Validator\Constraints\SegmentDateValidator::class,
+                'arguments' => [
+                    'mautic.lead.model.lead_segment_filter_factory',
+                    'translator',
+                ],
+                'tag'       => 'validator.constraint_validator',
+            ],
```

Comment:
Please do not add more service definition to the config.php files. I can see this one needs a tag but it can be done in the services.php. See https://github.com/mautic/mautic/blob/5.x/app/bundles/CoreBundle/Config/services.php#L55 for example.